misk-feature-testing/src/test/kotlin/misk/feature/testing/FakeFeatureFlagsModuleTest.kt

```diff
@@ -11,9 +14,15 @@ class FakeFeatureFlagsModuleTest {
   fun testModule() {
     val injector = Guice.createInjector(FakeFeatureFlagsModule().withOverrides {
       override(Feature("foo"), 24)
+    }, object : KAbstractModule() {
+      override fun configure() {
+        // Misk services automatically get this binding, but no need to depend on all of misk to
```

Comment:
"no need to depend on all of misk" is a recurring theme in my experience

Reply:
We will modularize Misk!